ocean_lib/assets/asset_downloader.py

```diff
@@ -84,4 +84,4 @@ def download_asset_files(
             i,
         )
 
-    return destination
+    return asset_folder, destination ### verify if asset_folder is a file => append to destination
```

Comment:
Ideally we don't change the interface, because that's a breaking change to the API. For that you normally need a major version change. Why not just give the path promised by the interface?